src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class BooleanExpression {
+
+    /**
+     * Реализуйте метод booleanExpression(boolean a, boolean b, boolean c, boolean d),
+     *          возвращающий true, если среди четырех его аргументов ровно два истинны (любые)
```

Comment:
make sure each line is not longer than 100 characters https://mate-academy.github.io/style-guides/java/java.html#s4.4-column-limit